jenkins-master/jobs/mozilla-central_update/config.xml

```diff
@@ -36,6 +36,11 @@
           <defaultValue></defaultValue>
         </org.jvnet.jenkins.plugins.nodelabelparameter.LabelParameterDefinition>
         <hudson.model.StringParameterDefinition>
+          <name>REPOSITORY</name>
+          <description>The repository of Firefox.</description>
+          <defaultValue>$REPOSITORY</defaultValue>
```

Comment:
Ugh! Not sure why this ended up with $REPOSITORY but that broke mozmill-ci. I will push a follow-up fix for that.